### PR TITLE
Handle order not found, #172

### PR DESCRIPTION
**Pull request by DarrenWestwood**
Handle error in case order not found by address, #172

### CODE REVIEW

File: php/WC_Gateway_Blockonomics.php [hunk 1]
```diff
@@ -196,8 +196,8 @@ public function check_blockonomics_callback()
         if ($callback_secret  && $callback_secret == $secret) {
             $addr = $_REQUEST['addr'];
             $order = $orders[$addr];
```
Comment (shivaenigma) [inline php/WC_Gateway_Blockonomics.php:198]: Giving undefined index on when looking up addr inside orders
`Notice: Undefined index: 18MKYd8YnVtsAakrTmpBm3qLfsXLdKjsqf in /home/enigma/blockonomics-bitcoin-payments/php/WC_Gateway_Blockonomics.php on line 198`

File: php/WC_Gateway_Blockonomics.php [hunk 2]
```diff
@@ -241,7 +241,11 @@ public function check_blockonomics_callback()
                     update_post_meta($wc_order->get_id(), 'expected_btc_amount', $order['satoshi']/1.0e8);
                 }
                 update_option('blockonomics_orders', $orders);
+            }else{
+                exit("Error: order not found");
             }
+        }else{
+            exit();
```
Comment (shivaenigma) [inline php/WC_Gateway_Blockonomics.php:248]: else and exit() here is uncessary as the function is anyway existing

File: php/WC_Gateway_Blockonomics.php (hunk 1 above)
Comment (DarrenWestwood) [reply php/WC_Gateway_Blockonomics.php]: Updated to check key exists - 2d63e48

File: php/WC_Gateway_Blockonomics.php (hunk 2 above)
Comment (DarrenWestwood) [reply php/WC_Gateway_Blockonomics.php]: Removed in 3a92048

**Pull request merged**